Modules/Exercise05/Tests/Feature/Http/Controllers/Exercise05ControllerTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Feature\Http\Controllers;
+
+use Modules\Exercise05\Http\Controllers\Exercise05Controller;
+use Modules\Exercise05\Http\Requests\OrderRequest;
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class Exercise05ControllerTest extends TestCase
+{
+    protected $orderService;
+    protected $exercise05Controller;
+
+    protected function setUp(): void
+    {
+        parent::setup();
+
+        $this->orderService = $this->mock(OrderService::class);
+        $this->exercise05Controller = new Exercise05Controller(
+            $this->orderService
```

Comment:
cho lên cùng dòng với bên trên đi bạn ơi , để dưới làm gì cho tốn line :D

Reply:
ở dưới dùng shoudReceive thì cho làm gì???

Reply:
```suggestion
            $this->exercise05Controller = new Exercise05Controller($this->orderService);
```
Ý anh VA là thế này thôi ợ, trông cho nó ngắn ấy mờ